db/database.go

```diff
@@ -818,6 +818,8 @@ func (db *DatabaseContext) DeleteUserSessions(userName string) error {
 // the document to accomplish the same result.
 func (db *Database) Compact() (int, error) {
 
+	ctx := db.Ctx
```

Comment:
There are other Admin API endpoints that might merit this treatment - why Compact in this case?

Reply:
I don't feel like it would add a lot to the others. Its more useful for compact in particular because it takes a while to run.